Assignment-clonecoding/Assignment-clonecoding/2nd/WelcomeViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  WelcomeViewController.swift
+//  Assignment-clonecoding
+//
+//  Created by 송여경 on 4/20/24.
+//
+
+import Foundation
+import UIKit
+
+class WelcomeViewController: UIViewController {
+    var id : String?
+    
+    private let welcomeLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont(name: "Pretendard-Bold", size: 23)
+        label.textAlignment = .center
+        label.textColor = UIColor(red: 214/255, green: 214/255, blue: 214/255, alpha: 1)
+        label.numberOfLines = 2
+        return label
+    }()
+    
+    private let logoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "Tiving")
+        return imageView
+        
+    }()
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = .black
+        setLayout()
+        updateWelcomeText()
+    }
+    
+    
+    private lazy var mainButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("메인으로", for: .normal)
+        button.backgroundColor = UIColor(red: 255/255, green: 20/255, blue: 60/255, alpha: 1)
+        button.setTitleColor(UIColor(red: 255/255, green: 255/255, blue: 255/255, alpha: 1), for: .normal)
+        button.titleLabel?.font = UIFont(name: "Pretendard-SemiBold", size: 18)
+        button.addTarget(self, action: #selector(goToMain), for: .touchUpInside)
+        return button
+    }()
+    
+    private func setLayout() {
+        [logoImageView,welcomeLabel,mainButton].forEach {
+            self.view.addSubview($0)
+            $0.translatesAutoresizingMaskIntoConstraints = false
+        }
+        
+        logoImageView.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalToSuperview().offset(58)
```

Comment:
현재 네비게이션바의 백버튼과 이미지가 겹치는 것 같습니다! 
```suggestion
            make.top.equalTo(view.safeAreaLayoutGuide.snp.top).offset(58)
``` 
네비게이션바와 로고가 겹치는 것을 해결하고 싶으면 이런식으로 쓰시는 것도 추천드립니다!

Reply:
감사합니다!!